spec/acceptance/vhost_spec.rb

```diff
@@ -1326,7 +1326,8 @@ class { 'apache': }
   end
 
   # So what does this work on?
```

Comment:
re that comment ^ what ever it works on, doesn't matter
you should be using mod_fcgid ;)